docs/docs/production/testing-your-assistant.mdx

```diff
@@ -35,511 +32,9 @@ If data validation results in errors, training a model can also fail or yield ba
-
-```yaml-rasa title="tests/test_stories.yml" {5,9,14,20}
-stories:
-- story: A test story with a form
-  steps:
-  - user: |
-      hi
-    intent: greet
-  - action: utter_greet
-  - user: |
-      im looking for a restaurant
-    intent: request_restaurant
-  - action: restaurant_form
-  - active_loop: restaurant_form
-  - user: |
-      [afghan](cuisine) food
-    intent: inform
-  - action: restaurant_form
-  - active_loop: null
-  - action: utter_slots_values
-  - user: |
-      thanks
-    intent: thankyou
-  - action: utter_no_worries
-```
-
-  </TabItem>
-  <TabItem value="formsunhappypath">
-
-```yaml-rasa title="tests/test_stories.yml" {5,9,14,21}
-stories:
-- story: A test story with unexpected input during a form
-  steps:
-  - user: |
-      hi
-    intent: greet
-  - action: utter_greet
-  - user: |
-      im looking for a restaurant
-    intent: request_restaurant
-  - action: restaurant_form
-  - active_loop: restaurant_form
-  - user: |
-      How's the weather?
-    intent: chitchat
-  - action: utter_chitchat
-  - action: restaurant_form
-  - active_loop: null
-  - action: utter_slots_values
-  - user: |
-      thanks
-    intent: thankyou
-  - action: utter_no_worries
-```
-
-  </TabItem>
-  <TabItem value="entities">
-
-```yaml-rasa title="tests/test_stories.yml" {5,9,13}
-stories:
-- story: A basic test story with multiple entities for a single token
-  steps:
-  - user: |
-      hello
-    intent: greet
-  - action: utter_ask_howcanhelp
-  - user: |
-     show me [chinese]{"entity": "cuisine"} restaurants
-    intent: inform
-  - action: utter_ask_location
-  - user: |
-      in [Paris][{"entity": "location"}, {"entity": "city"}]
-    intent: inform
-  - action: utter_ask_price
-```
-
-  </TabItem>
-</Tabs>
-
-By default, the command will run tests on stories from any files with names starting with `test_`. You can also provide
-a specific test stories file or directory with the `--stories` argument.
-You can test your assistant against them by running:
-
-```bash
-rasa test
-```
-
-Conversation testing is only as thorough and accurate as the test
-cases you include, so you should continue to grow your set of test stories
-as you make improvements to your assistant. A good rule of thumb to follow is that you should aim for your test stories
-to be representative of the true distribution of real conversations.
-
-See the [CLI documentation on `rasa test`](../command-line-interface.mdx#rasa-test) for
-more configuration options.
-
-:::caution Testing Custom Actions
-[Custom Actions](../concepts/custom-actions.mdx) are not executed as part of test stories. If your custom
-actions append any events to the conversation, this has to be reflected in your test story
-(e.g. by adding `slot_was_set` events to your test story).
-
-To test the code of your custom actions, you should write unit tests
-for them and include these tests in your [CI/CD pipeline](./setting-up-ci-cd.mdx).
-
-:::
-
-## Evaluating an NLU Model
```

Comment:
I think this will still be relevant to some extent when we have clarity on how CALM works with NLU pipeline. For now I decided to remove everything (the content is still present under `./building-classic-assistant`), because I think the content will be different, as the use cases will be different.

Reply:
yes that makes sense and I agree